src/lib.rs

```diff
@@ -10,23 +10,9 @@
 #[macro_use]
 pub extern crate juniper;
 
-pub use juniper::{Arguments, ExecutionResult, Executor, Object, Value};
-
-pub use client::WarpgrapherClient;
 pub use error::Error;
 pub use error::ErrorKind;
 
-pub use engine::config::{
-    WarpgrapherConfig, WarpgrapherEndpoint, WarpgrapherResolverFunc, WarpgrapherResolvers,
-    WarpgrapherType, WarpgrapherValidatorFunc, WarpgrapherValidators,
-};
-pub use engine::context::{GraphQLContext, WarpgrapherRequestContext};
-pub use engine::extensions::{Extension, WarpgrapherExtensions};
-pub use engine::neo4j::Neo4jEndpoint;
-pub use engine::objects::Node;
-pub use engine::schema::{Info, Property};
-pub use engine::Engine;
```

Comment:
I'd be okay with re-exporting engine::config::Config and engine::Engine, given that if I remember correctly, every client that uses Warpgrapher has to use those two types.  Other than that, the drastic cutting down of re-exports looks good.  (And I could be talked out of the two above, if you'd rather not.)

Reply:
I agree with that suggestion.